docs/operate/reference/components/motor/dmc4000.md

```diff
@@ -9,8 +9,14 @@ aliases:
   - "/components/motor/dmc4000/"
 component_description: "Stepper motor driven by a DMC-40x0 series motion controller."
 # SMEs: Rand, James
+toc_hide: true
 ---
 
+{{% hiddencontent %}}
+A private module exists that supports the DMC4000 motor controller.
+If you would like to use this controller with Viam, contact Viam support.
+{{% /hiddencontent %}}
+
```

Comment:
I see. So with `toc_hide` it'll still end up in typesense which probably is ok. Since if people search for it they can find the page and then know to contact support. For that case I'd actually make this content not hidden:

```suggestion
{{< alert title="Note" color="note" >}}
{{% hiddencontent %}}
A private module exists that supports the DMC4000 motor controller.
If you would like to use this controller with Viam, contact [Viam support](mailto:support@viam.com).
{{% /hiddencontent %}}
{{< /alert >}}

```

Reply:
Your suggestion keeps the content hidden but with an alert around it :) I will apply the intended suggestion including the support email.
```suggestion
{{< alert title="Note" color="note" >}}
A private module exists that supports the DMC4000 motor controller.
If you would like to use this controller with Viam, contact [Viam support](mailto:support@viam.com).
{{< /alert >}}

```